Jetchat/app/src/main/java/com/example/compose/jetchat/DroidconEmbeddingsWrapper.kt

```diff
@@ -296,17 +299,20 @@ class DroidconEmbeddingsWrapper(val context: Context?) {
             Log.i("LLM", "Top match was ${sortedVectors.lastKey()} which was below 0.8 and failed to meet criteria for grounding data")
         }
 
-        // ALWAYS add the date and time to every prompt
-        var date = Constants.TEST_DATE
-        var time = Constants.TEST_TIME
-        if (date == "") {
-            date = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"))
-        }
-        if (time == "") {
-            time  = LocalDateTime.now().format(DateTimeFormatter.ofPattern("HH:mm"))
+        if (messagePreamble.isNullOrEmpty()) {
+            // ONLY show date/time when embeddings are empty, as it triggers the SessionsByTime function (I THINK)
+            // ALWAYS add the date and time to every prompt
```

Comment:
```suggestion
            // ONLY show date/time when embeddings are empty, as it triggers the SessionsByTime function
```

hoping the "i think" has been resolved 😜

Reply:
🤔 not sure!